src/Symfony/Component/Console/Question/Question.php

```diff
@@ -150,10 +147,8 @@ public function setAutocompleterValues($values)
             $callback = static function () use ($values, &$valueCache) {
                 return $valueCache ?? $valueCache = iterator_to_array($values, false);
             };
-        } elseif (null === $values) {
-            $callback = null;
         } else {
-            throw new InvalidArgumentException('Autocompleter values can be either an array, "null" or a "Traversable" object.');
+            $callback = null;
```

Comment:
branch

Reply:
I'm removing a type check here, that was obsoleted by the new parameter type declaration. Lower branches still need the type-check.